### PR TITLE
Infura Network response tests

### PR DESCRIPTION
**Pull request by tmashuang**
Redo Infura Controller Test

### CODE REVIEW

File: test/unit/infura-controller-test.js [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
-//
-//       })
-//     })
-//   })
-// })
+const assert = require('assert')
+const InfuraController = require('../../app/scripts/controllers/infura')
+
+describe('infura-controller', function () {
+  var infuraController
```
Comment (frankiebee) [inline test/unit/infura-controller-test.js]: use let

File: test/unit/infura-controller-test.js [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
+  let response
+
+  before(async function () {
+    infuraController = new InfuraController()
+    response = await infuraController.checkInfuraNetworkStatus()
```
Comment (frankiebee) [inline test/unit/infura-controller-test.js]: are you actually fetching from the network?

Comment (tmashuang) [inline test/unit/infura-controller-test.js]: yay or nay?

Comment (frankiebee) [inline test/unit/infura-controller-test.js]: no please stub and test the methods

Comment (flyswatter) [inline test/unit/infura-controller-test.js]: I agree, you probably should mock the network response. I know we hit the network in other tests, but that's bad, so it's worth figuring it out and doing it right.

**Review by kumavis — APPROVED** [review]

**Pull request merged**